Tests/SkillFunctionalTests/appsettings.json

```diff
@@ -7,6 +7,7 @@
   },
 
   "TestRequestTimeout": 180,
+  "ThinkTime":  500,
```

Comment:
This could be confusing because `TestRequestTimeout` is handled in seconds and `ThinkTime` in milliseconds, we should decide to use one or the other, now in the future should we use ms or sec, I'll prefer ms, since most to not say all libraries/functionality use ms.